assets/locales/en.json

```
@@ -238,5 +240,14 @@
     "message": "Ongoing operation is taking more resources than expected. Do you wish to abort, and forcefully reload the interface?",
     "forceReload": "Yes, reload",
     "doNothing": "Do nothing"
+  },
+  "protocolHandlerDialog": {
+    "title": "Opening IPFS URIs",
```

I recently realized humans have no idea what URI is  :robot: 
```suggestion
    "title": "Opening IPFS address",
```